### ISSUE TITLE
New warning when upgrading from 5.6.1 to 6.0.0

### ISSUE THREAD

**Issue by ericcornelissen** [stryker-mutator/stryker-js#3497]
**Question**

When upgrading from version 5.6.1 to version 6.0.0 I started seeing a new warning:

```log
> stryker run stryker.config.json

WARN PluginLoader Expression "@stryker-mutator/*" not resulted in plugins to load.
INFO InputFileResolver Found 1 of 50 file(s) to be mutated.
...
```

Any idea what is causing this warning?

**Stryker environment**

```
+-- @stryker-mutator/core@6.0.0
```

```
+-- uvu@0.5.3
```

**Additional context**

What I tried:

- I didn't see anything obviously related to this in [the release notes for 6.0.0](https://github.com/stryker-mutator/stryker-js/releases/tag/v6.0.0).
- I tried re-installing `@stryker-mutator/core` from scratch to update transitive dependencies, this didn't help.

The project:

- The project were I'm seeing this is [git-tag-annotation-action](https://github.com/ericcornelissen/git-tag-annotation-action).
  - I'm not using any plugins.   
  - [Here's the Stryker configuration file](https://github.com/ericcornelissen/git-tag-annotation-action/blob/950f9d8e60124b1bf62e87faea5aa936dda24e05/stryker.config.json).
- The `@stryker-mutator/core` dependency is being updated in [this Pull Request](https://github.com/ericcornelissen/git-tag-annotation-action/pull/179).
- Here are [the CI logs](https://github.com/ericcornelissen/git-tag-annotation-action/runs/6286171676?check_suite_focus=true#step:7:11) after only upgrading the `@stryker-mutator/core` dependency.
- Here are [the CI logs](https://github.com/ericcornelissen/git-tag-annotation-action/runs/6294285375?check_suite_focus=true#step:7:10) after re-installing the `@stryker-mutator/core` dependency.
- Here are [CI logs](https://github.com/ericcornelissen/git-tag-annotation-action/runs/6260890660?check_suite_focus=true#step:5:10) of the latest run prior to upgrading to 6.0.0.

**Comment by nicojs** [comment]
Wow! Thanks for upgrading so fast and providing feedback! As a workaround, you can simply ignore this warning.

Part of 6.0.0 was a rewrite of the plugin loader (it now uses `import` instead of `require`). It isn't directly mentioned in the v6.0.0 changelog, but it is buried somewhere in [the v6.0.0-beta.0 release notes](https://github.com/stryker-mutator/stryker-js/releases/tag/v6.0.0-beta.0): #3409

**Comment by nicojs** [comment]
I've just released v6.0.1 with a fix 🤗

**Comment by ericcornelissen** [comment]
Thanks for the quick fix @nicojs :clap:
